### ISSUE TITLE
Cannot use Auth SDK

### ISSUE THREAD

**Issue by wtrocki** [aerogear/aerogear-android-sdk#277]
## Expected Behavior

Code compiles

## Actual Behavior
Error:
```
Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for <appAuthRedirectScheme> is provided. *
```

## Steps to Reproduce the Problem

  1. Follow documentation on https://docs.aerogear.org/aerogear/latest/identity-management.html
  
## Specifications

  - Core SDK Version: 2.0.0
  - Relevant Service Modules: AuthSDK

**Comment by wtrocki** [comment]
See also: https://github.com/openid/AppAuth-Android/issues/128
Our SDK is using hardcoded value: https://github.com/aerogear/aerogear-android-sdk/blob/master/auth/build.gradle#L11

**Comment by wtrocki** [comment]
https://github.com/aerogear/aerogear-android-sdk/pull/278

**Comment by trepel** [comment]
@wtrocki I have noticed that with the change from #278 the task `:auth:processDebugAndroidTestManifest` is not working anymore. Not sure if this is an issue or not.

**Comment by wtrocki** [comment]
@trepel Yes.. Actually this was put there only for purpose of the tests/local development but was crashing production usages. I have restored that to placeholder now in https://github.com/aerogear/aerogear-android-sdk/pull/283 After this is going to be merged everything should be back to normal.
